lib/config/error-codes/index.js

```diff
@@ -85,8 +91,27 @@ function checkErrorCodes(errorCodes) {
         assert(Number.isInteger(errorName.code), `Error configuration file : Field 'code' must be an integer in '${domain}-${subdomain}-${error}'.`);
         assert(errorName.code > 0x0000 && errorName.code <= 0xFFFF, `Error configuration file : Field 'code' must be between 0 and 65535 in '${domain}-${subdomain}-${error}'.`);
       }
+      assert(isUnique(codesErrors), `Error configuring file : You have an error code duplicate in ${domain}-${subdomain}.`);
+      codesErrors.length = 0;
     }
+    assert(isUnique(codesSubdomains), `Error configuring file : You have a subdomain code duplicate in ${domain}.`);
+    codesSubdomains.length = 0;
   }
+  assert(isUnique(codesDomains), 'Error configuring file : You have a domain code duplicate.');
+  codesDomains.length = 0;
+}
+
+function isUnique(codesArray) {
```

Comment:
```suggestion
function isUnique(arr) {
  return arr.filter((itm, i) => return arr.lastIndexOf(itm) === i && arr.indexOf(itm) !== i).length === 0;
}
```

Reply:
Smaller implementation :slightly_smiling_face:

Reply:
I did it another way with lodash